specification.md

```diff
@@ -0,0 +1,131 @@
+# Feature Driven Architecture
+
+Feature Driven Architecture (FDA) is a structural framework that helps organize the codebase for visual interfaces in a way that lets you keep a constant development velocity while the amount of Features is growing.
+
+## Rationale
+
+We have various component models that let us compose the UI. We have multiple data flow models that simplify working with the data. And yet, we struggle to build an application that allows many people to work in parallel. We suffer an exponential slow-down when the codebase grows.
+
+The reason is the lack of agreement on splitting an application in a way that allows more people to work in parallel and keep the complexity under control.
+
+FDA is the missing piece that builds **on top** of existing component and data-flow models.
+
+Instead of organizing code by data and abstraction types, we want to organize the code based on the user-facing features that solve a user problem. This way, we believe it is easier to scale, communicate and navigate the codebase.
+
+## Goals
```

Comment:
Could just be semantics but these seem more like benefits of conforming to the architecture as opposed to goals?

Reply:
I agree that these are also benefits as much as they are goals. What do you think is a difference?